scripts/mbedtls_dev/bignum_mod_raw.py

```diff
@@ -40,6 +43,385 @@ class BignumModRawTarget(test_data_generation.BaseTarget, metaclass=ABCMeta):
 # END MERGE SLOT 4
 
 # BEGIN MERGE SLOT 5
+class BignumModRawOperation(bignum_common.OperationCommon, BignumModRawTarget, metaclass=ABCMeta):
+    #pylint: disable=abstract-method
+    """Common features for bignum mod raw operations."""
+    modulus_input_cases = [
+        ("3", "", [('1', '1'), ('2', '2'), ('1', '2')]),
+        ("7", "", [('4', '4'), ('1', '1'), ('5', '5'), ('1', '6')]),
+        ("B", "", [('4', '6'), ('a', '9'), ('2', '6'), ('2', '5')]),
```

Comment:
Questions:

* How is that data generated?
* How is it meant to be consumed?
* Is it meant to be mod_raw specific or it can be used by other modules?
* Why not use pep8 to format that data?

My proposal is to move that to its own object, either in module or in coomon, and set it on the classes using is using `settattrib` using a commonly accepted name eg. `setattr(self, 'input_moduli/input_cases/etc', bignum_common.common_test_cases)`

Also add a comment, on what each member is used, or use namedtuple or dictionaries to add the context the the accessing label. 

Consuming or modifying those data structures is not implicit for the new users.